mesonbuild/interpreter/interpreter.py

```diff
@@ -23,7 +23,7 @@
                         extract_as_list, has_path_sep, path_is_in_root, PerMachine)
 from ..options import OptionKey
 from ..programs import ExternalProgram, NonExistingExternalProgram
-from ..dependencies import Dependency
+from ..dependencies import Dependency, pkgconfig
```

Comment:
It's desirable to avoid too many imports at startup and this is rarely used, in exactly one location. Maybe this import can be moved down to inside `if name == 'pkg-config'`?